container/fs/fs.go

```diff
@@ -200,17 +206,38 @@ func (self *RealFsInfo) GetFsInfoForPath(mountSet map[string]struct{}) ([]Fs, er
 }
 
 // GetFsStats returns filesystem statistics for a given container
-func GetFsStats(container *docker.Container) (map[string]wrapper.FilesystemInterface, error) {
+func (du *DiskUsageCollector) GetStats(stats *container.Statistics, opts container.GetStatOpt) error {
```

Comment:
Right now `GetStats()` routine is basing on a constant path to storage mount point, `/var/lib/docker`. It might be good idea to discover this path while initializing docker client with call to `GetDockerParam("DockerRootDir")` - see Docker API [`/info` endpoint](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/display-system-wide-information).